spark/src/main/scala/io/delta/sharing/spark/DeltaSharingSource.scala

```diff
@@ -952,7 +952,13 @@ case class DeltaSharingSource(
     new AdmissionLimits().toReadLimit
   }
 
+  def debug(str: String): Unit = {
+    // scalastyle:off println
+    Console.println(s"----[linzhou]----SharingSource: $str")
```

Comment:
should we turn the style back on after print?

Consider printing "DeltaSharingSourceDebug" instead on linzhou?